### DISCUSSION TITLE
Doesn’t accept school documents

### DISCUSSION THREAD

**Discussion by ARLBR10** [community/community#56227] [GitHub Education]
### Hi everyone,

Hello so I am trying to enter to GitHub for education by a month and the program says that can’t now if I’m a really a student but I already have sends my school ID, payment for the school and keep saying to take photo with the cellphone but I already took.

if someone can help me I will be thankful

![IMG_0473](https://github.com/community/community/assets/71902040/374c6744-9212-48f9-8d4c-d0ecc238a8c1)

**Comment by samus-aran** [comment]
Hi @ARLBR10 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status. We also suggest you provide a live photo rather than uploading an image. 

Hope this helps 👍
